s-ramp-ide/srampRepositoryBrowser/src/main/java/cz/muni/fi/srampRepositoryBrowser/manager/ArtifactTypeGuessingService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2013 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.muni.fi.srampRepositoryBrowser.manager;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.overlord.sramp.common.ArtifactTypeEnum;
+
+/**
+ * Service used to guess artifact types from filenames.
+ * 
+ * @author eric.wittmann@redhat.com
+ */
+
+public class ArtifactTypeGuessingService {
```

Comment:
This can be completely removed.  We now automatically detect the type, server-side, if one is not provided.  So, no need to automatically detect the type from a client.